packages/serverless-component/src/component.ts

```diff
@@ -42,9 +43,19 @@ class NextjsComponent extends Component {
     );
   }
 
+  readRoutesManifest(nextConfigPath: string): Promise<RoutesManifest> {
+    return readJSON(join(nextConfigPath, ".next/routes-manifest.json"));
+  }
+
+  pathPattern(pattern: string, routesManifest: RoutesManifest): string {
+    const basePath = routesManifest.basePath;
+    return basePath.length === 0 ? pattern : `${basePath.slice(1)}/${pattern}`;
```

Comment:
Is this backward compatible with an older route manifest that doesn't include basePath? i.e. what if basePath is undefined?

Reply:
Added an extra check for this :+1: